### ISSUE TITLE
Locks don't render - Cannot open/break chest with key.

### ISSUE THREAD

**Issue by Succubism** [VazkiiModsArchive/Charset#24]
Using:
- Proton Pack v1.0.4
- Charset-0.3.0-pre15

Chests locked with the "Lock" item do not render lock on chest's face.
Locked chest cannot be unlocked with corrosponding Key or broken.

[Screencap](http://i.imgur.com/motwnhN.png)

**Comment by palaster** [comment]
Are you playing on a server or in single player?

**Comment by Succubism** [comment]
Playing on Vazkii's Patreon server

**Comment by palaster** [comment]
Okay, then I am going to have to test it on a server, because I haven't been able to reproduce the issue on my side.

**Comment by palaster** [comment]
Okay, I have reproduced the issue it seems to only happen on servers.

**Comment by SmallSansSerif** [comment]
Yes experiencing this on a server too. Any hope of a fix? It seems to be any type of chest as well as in different wood types from Quark don't make any difference.

**Comment by SmallSansSerif** [comment]
Also would love to see the lock have iron chest support as well.

**Comment by palaster** [comment]
I finally got a fix for it. https://github.com/Vazkii/Charset/pull/26
It's still a bit finicky, but it works. One thing you will notice is that on  servers is that there is a small delay from when you place the lock to when it will appears on the inventory. Another, thing is their is a small graphical glitch which might allow you to see through the block (only seen with normal chest so far).

**Comment by SmallSansSerif** [comment]
So this fixes keys not working as well to unlock?

**Comment by palaster** [comment]
The issue had nothing to do with the key, the issue was that the locks weren't actually there on servers.
